README.md

```diff
@@ -111,7 +111,7 @@ chw_flow_sensor.type_docs() # 'Sensor which measures the volumetric flow of chil
 chw_flow_sensor.type_uri() # 'https://project-haystack.org/def/phIoT/3.9.10#chilled-water-flow-sensor'
 
 # Generate an id for the point and bind it to a namespace
-chw_flow_sensor.gen_uuid() # UUID('2733f091-be5b-4983-b701-b7e42c52b72c')
+chw_flow_sensor.set_id() # UUID('2733f091-be5b-4983-b701-b7e42c52b72c')
```

Comment:
Why did we make this change?

Reply:
`gen_uuid` didn't work for me because I think it was renamed...

https://github.com/NREL/tasty/blob/25dce9fe2d57120917b68db4229cd221809f1a03/tasty/entities.py#L195-L199

Reply:
ah 👍